commands/safe_backup.go

```diff
@@ -9,13 +9,15 @@ import (
 )
 
 func init() {
+	// TODO: This command is deprecated and will be removed soon.
 	rootCmd.AddCommand(&cobra.Command{
 		Use:   "backup",
 		Short: "Backup the safe",
 		Long:  "The backup command is used to backup your current safe.",
 		Run: func(cmd *cobra.Command, args []string) {
-			runCommand(Backup, cmd, args)
+			runMovedCommand(Backup, cmd, args, "safe backup")
```

Comment:
This is a nice pattern for deprecations; perhaps we can call it `runDeprecatedCommand` and use this pattern going forward?

Reply:
That's a totally valid point, LGTM.